Lombiq.Tests.UI/Services/UITestExecutionSession.cs

```diff
@@ -221,16 +227,33 @@ private Exception PrepareAndLogException(Exception ex)
         return ex;
     }
 
-    private async Task CreateFailureDumpAsync(
-        Exception ex,
+    private async Task FailureTestDumpProcessAsync(string dumpContainerPath, Exception ex)
+    {
+        if (_context == null) return;
+
+        var debugInformationPath = GetDebugInformationPath(dumpContainerPath);
+
+        if (_context.IsBrowserRunning) await CaptureBrowserUsingDumpsAsync(debugInformationPath);
+
+        if (_dumpConfiguration.CaptureAppSnapshot) await CaptureAppSnapshotAsync(dumpContainerPath);
+
+        CaptureMarkupValidationResults(ex, debugInformationPath);
```

Comment:
Each line in this method is separated with a blank line, which doesn't seem readable.
Can we group the `Capture*()` method calls together?
```suggestion
        if (_context.IsBrowserRunning) await CaptureBrowserUsingDumpsAsync(debugInformationPath);
        if (_dumpConfiguration.CaptureAppSnapshot) await CaptureAppSnapshotAsync(dumpContainerPath);
        CaptureMarkupValidationResults(ex, debugInformationPath);
```

Reply:
Yeah, better like that.